upgrade.sh

```diff
@@ -67,6 +67,28 @@ undeploy -n gatekeeper-system gatekeeper-constraints
 undeploy -n gatekeeper-system gatekeeper-policy-library
 undeploy -n gatekeeper-system gatekeeper
 
+# Checks for gatekeeper-system namespace on the system.
```

Comment:
Hi @jeremy-duckworth, From the 1.4 update logs that are posted from QE team, these changes may not be required gatekeeper-system namespace is not available after upgrade. As a safe side we can include namespace removal if its found. Would need your inputs here. Thank you.

Reply:
@yarlagadda-srinivasa-rao Looking at 1.3.1 install inputs, I clearly see a version of dry-dock that contains creation of the opa-gatekeeper namespace:

https://github.com/Cray-HPE/csm/blob/v1.3.1/manifests/platform.yaml#L40
https://github.com/Cray-HPE/cray-drydock/blob/8c60d0a969698b77426003abb34d07d7564dff56/kubernetes/cray-drydock/templates/namespaces.yaml#L108 (looking at version and release tag)

Thus, I don't know what would cleaned up the namespace during an upgrade from 1.3.1 to 1.4.0. Looking at the test system, the API pod logs don't go back far enough to see if the ns was cleaned up another way (or created on install of 1.3.1).

Reply:
Yes @jeremy-duckworth, safe side we will keep the namespace removal code. This will remove the namespace only if found during upgrade. Could you please review and approve the change? Thank you

Reply:
Sure, just pointing out that there is probably a flaw in the test method. I don't have an issue with the removal code. This actually needs to be tested on a system.

Reply:
Received slack notice that this has been sorted...